waku/waku_store/protocol.nim

```diff
@@ -99,15 +84,33 @@ proc handleQueryRequest*(
   return res.encode().buffer
 
 proc initProtocolHandler(self: WakuStore) =
+  let rejectReposnseBuffer = StoreQueryResponse(
+    ## We will not copy and decode RPC buffer from stream only for requestId
+    ## in reject case as it is comparably too expensive and opens possible
+    ## attack surface
+    requestId: "N/A",
+    statusCode: uint32(ErrorCode.TOO_MANY_REQUESTS),
+    statusDesc: $ErrorCode.TOO_MANY_REQUESTS,
```

Comment:
I repeated the comment below. I wonder if we could enrich the response by indicating what is the actual limit

Reply:
Do you mean in the response for the client? I don't want to expose it to any malicious peer to take advantage on it. WDYT?